modules/post-installation-adding-nutanix-failure-domains-control-planes.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/adding-nutanix-failure-domains.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="post-installation-adding-nutanix-failure-domains-control-planes_{context}"]
+= Distributing control planes across failure domains
+
+You distribute control planes across Nutanix failure domains by modifying the control plane machine set custom resource (CR).
```

Comment:
```suggestion
You distribute control planes across Nutanix failure domains by modifying the control plane machine set custom resource definition (CRD).
```

Reply:
This was raised early as part of an SME review. The decision was to go with CR.

https://github.com/openshift/openshift-docs/pull/69618#discussion_r1457598396